Library/Homebrew/extend/ARGV.rb

```diff
@@ -195,7 +195,7 @@ def force_bottle?
   # eg. `foo -ns -i --bar` has three switches, n, s and i
   def switch?(char)
     return false if char.length > 1
-    options_only.any? { |arg| arg[1, 1] != "-" && arg.include?(char) }
+    options_only.any? { |arg| arg.scan("-").size == 1 && arg.include?(char) }
```

Comment:
I realize I'm a bit late, but wouldn't `arg.rindex("-") == 0` more efficiently achieve the same task? `String#scan` seems like a sledgehammer in comparison …